api/C/include/csi_csr_access.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * Low-Level CSR ACCESS API
+ *
+ * APIs giving access to CSRs.
+ *
+ * Copyright (c) RISC-V International 2023. Creative Commons License. Auto-
+ * generated file: DO NOT EDIT
+ */
+
+#ifndef CSI_CSR_ACCESS_H
+#define CSI_CSR_ACCESS_H
+
+#include "csi_types.h"
+#include <stdint.h>
+
+/*
+ * macros
+ */
+#define __STR(s)                #s
+#define STRINGIFY(s)            __STR(s)
+
```

Comment:
Please define an extra macro `#define __ASM                        __asm`, which is used in below macro blocks